.cargo/config.toml

```diff
@@ -1,3 +1,7 @@
+# https://github.com/flamegraph-rs/flamegraph
+[target.x86_64-unknown-linux-gnu]
```

Comment:
should I have used `[target.'cfg(target_os = "linux")']`? due to different targets in GH action release.yaml? Perhaps also the linker should be explicit..